client/src/pages/trainee/course/[id]/index.tsx

```diff
@@ -0,0 +1,110 @@
+import ProgressPercentage from '@/src/shared/components/ProgressPercentage';
+import Breadcrumbs from '@/src/shared/components/Breadcrumbs';
+import type { Course, DBCourse } from '@/src/shared/utils';
+import Container from '@/src/shared/layouts/Container';
+import Button from '@/src/shared/components/Button';
+import Checkbox from '@mui/material/Checkbox';
```

Comment:
Maybe you can use the `<input type="checkbox"/>` and customize its class instead of installing a new library.